angularSTEP/src/app/current-profile-page/current-profile-page.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="profile-header">
+  <img class="profile-pic" src="{{ user.picUrl }}" alt="Profile pic for {{ user.displayName }}">
+  <div class="name-header">
+    <h1>{{user.displayName}}</h1>
+    <h3>@{{user.username}}</h3>
+  </div>
```

Comment:
Please be consistent on spaces between braces and what's going to be interpolated. 
`{{ user.displayName }}` vs `{{user.displayName}}`

Reply:
Oh yes, I am often sloppy when it comes to consistency. Thank you for catching that and I will definitely try to work on it.